validate_hif.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+from collections import defaultdict
+
+# network parameters
+filename = sys.argv[1]
+
+with open(filename) as file:
+    # load JSON file
+    data = json.loads(file.read())
+
+# check that keys do not deviate from the standard field names
+fields = {"network-type", "metadata", "nodes", "edges", "incidences"}
+if not set(data).issubset(fields):
```

Comment:
the same issue with graceful error handling. And a *library* version is definitely required

Reply:
Fixed.